packages/x-charts/src/hooks/useTicks.ts

```diff
@@ -45,6 +45,11 @@ export interface TickParams {
    * @default 'middle'
    */
   tickLabelPlacement?: 'middle' | 'tick';
+  /**
+   * The space between ticks when using an ordinal scale. It defines the minimum distance in pixels between two ticks.
+   * @default 0
```

Comment:
Do we want to increase the default in v9? Linear scales indirectly use 50 for this (because it calculates the default tick number as `range[1] - range[0] / 50`)